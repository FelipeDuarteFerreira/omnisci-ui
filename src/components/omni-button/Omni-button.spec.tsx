import React from "react"
import {render} from "enzyme"

import OmniButton from "./Omni-button"

test("Should render a button", () => {
  const props = {}
  const component = render(<OmniButton {...props} />)

  expect(component).toMatchSnapshot()
})
