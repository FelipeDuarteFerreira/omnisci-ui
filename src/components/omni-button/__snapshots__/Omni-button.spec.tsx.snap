// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a button 1`] = `
<button
  class="mdc-button omni-button"
>
  <span
    class="mdc-button__label"
  />
</button>
`;
